### DISCUSSION TITLE
Atlantis: Datatable frozen columns are transparent looks ugly when screen is scrolled

### DISCUSSION THREAD

**Discussion by rahulAnand1999** [primefaces/community#2461] [PrimeReact Templates] [answered]
<img width="1440" alt="Screenshot 2024-08-02 at 6 10 51 PM" src="https://github.com/user-attachments/assets/803d29c9-bd26-4489-9d70-2c025c66b4e4">

**Comment by melloware** [comment]
Weird I have Atlantis running on PrimeFaces 14.0.4 but I don't have a SubHeader Grouping example in my Table?  

I also just downloaded Atlantis 6.0.0 from my PrimeStore account and confirmed there is no SubHeader Grouping example?

**Reply by melloware** [reply]
Oh sorry this is for PrimeReact not PrimeFaces.  Looks like its an issue on their showcase as well: https://atlantis.primereact.org/uikit/table

**Comment by cagataycivici** [comment]
I'll check with the team, thank you for reporting it.

**Reply by rahulAnand1999** [reply]
Thanks, but any target when can we have this bug fixed?

**Reply by nitrogenous** [reply, marked as answer]
> Thanks, but any target when can we have this bug fixed?

Thank you for your patience. We are still working on the issue and expect to release a new version by next week

**Reply by rahulAnand1999** [reply]
hi @nitrogenous . When will be the new release taking place?

**Reply by nitrogenous** [reply]
Hi @rahulAnand1999 thank you for your patience. We are currently addressing a more complex issue that requires our immediate attention. We expect to release the new version within this week

**Comment by nitrogenous** [comment]
I discussed with my colleagues, and unfortunately, we won't be able to update our design at this time. However, you can achieve the desired effect by modifying the following lines in your files:

`public/theme/theme-light/_variables.scss` and `public/theme/theme-dark/_variables.scss`

```
$tableHeaderBg: transparent !default;
$tableHeaderCellBg: transparent !default;
$tableBodyRowBg: transparent !default;
$tableBodyRowEvenBg: transparent !default;
$tableFooterCellBg: transparent !default;
$tableFooterBg: transparent !default;
```

Feel free to reach out if you need further assistance!
